src/style.css

```
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 300px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300px;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+  color: #3e4e61;
+}
+
+.big {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  padding: 23px 20px 23px 62px;
+  background-position: left 25px bottom 25px;
+  background-size: 19px;
+  background-image: url(/images/Search.svg);
+}
+
+.small {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  padding: 10px 20px 11px 33px;
+  background-position: left 12px bottom 15px;
+  background-size: 11px;
+  background-image: url(/images/Search.svg);
+}
+
+.searchbar {
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+  outline: none;
+  font-family: Avenir, sans-serif;
```

use inherit value for inputs
```suggestion
  font-family: inherit;
```